src/App.js

```diff
@@ -1,13 +1,67 @@
 import './App.css';
+
+import { useState, createContext } from 'react';
+import { BrowserRouter, Routes, Route } from "react-router-dom"
```

Comment:
Maybe I should have known this from class and havent watched enough, but is this just to change the URL when you click the buttons?